### PR TITLE
delete joinTime from the response of create position

### PR DESCRIPTION
**Pull request by ValarDragon**
## What is the purpose of the change

Deletes joinTime from the response of create position. This is just extra boilerplate, its not doing anything.

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note

  - [ ] Does this pull request introduce a new feature or user-facing behavior changes?
  - [ ] Changelog entry added to `Unreleased` section of `CHANGELOG.md`?

Where is the change documented? 
  - [ ] Specification (`x/{module}/README.md`)
  - [ ] Osmosis documentation site
  - [ ] Code comments?
  - [ ] N/A

### CODE REVIEW

File: x/concentrated-liquidity/lp.go [hunk 1]
```diff
@@ -31,50 +31,50 @@ const noUnderlyingLockId = uint64(0)
 // - the liquidity delta is zero
 // - the amount0 or amount1 returned from the position update is less than the given minimums
 // - the pool or user does not have enough tokens to satisfy the requested amount
-func (k Keeper) createPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, tokensProvided sdk.Coins, amount0Min, amount1Min sdk.Int, lowerTick, upperTick int64) (positionId uint64, actualAmount0 sdk.Int, actualAmount1 sdk.Int, liquidityDelta sdk.Dec, joinTime time.Time, lowerTickResult int64, upperTickResult int64, err error) {
+func (k Keeper) createPosition(ctx sdk.Context, poolId uint64, owner sdk.AccAddress, tokensProvided sdk.Coins, amount0Min, amount1Min sdk.Int, lowerTick, upperTick int64) (positionId uint64, actualAmount0 sdk.Int, actualAmount1 sdk.Int, liquidityDelta sdk.Dec, lowerTickResult int64, upperTickResult int64, err error) {
```
[the remark this answers is not in the document]
Comment (ValarDragon) [reply x/concentrated-liquidity/lp.go:34]: Agreed, right now this feels quite untenable.

File: x/superfluid/keeper/msg_server.go [hunk 2]
```diff
@@ -221,7 +221,6 @@ func (server msgServer) UnlockAndMigrateSharesToFullRangeConcentratedPosition(go
 			sdk.NewAttribute(types.AttributeAmount0, amount0.String()),
 			sdk.NewAttribute(types.AttributeAmount1, amount1.String()),
 			sdk.NewAttribute(types.AttributeLiquidity, liquidity.String()),
-			sdk.NewAttribute(types.AttributeJoinTime, joinTime.String()),
```
Comment (ValarDragon) [inline x/superfluid/keeper/msg_server.go:224]: Removed join time from this superfluid event. (As its always ctx.BlockTime)

**Review by p0mvn — APPROVED** [review]
LGTM

**Comment by p0mvn** [comment]
Merging with one ACK since the change is simple

**Pull request merged**